src/types.ts

```diff
@@ -69,8 +70,8 @@ export type ServiceProviderSettings = {
   isAssertionEncrypted?: boolean;
   encPrivateKey?: string | Buffer;
   encPrivateKeyPass?: string | Buffer;
-  assertionConsumerService?: Array<{ Binding: string, Location: string }>;
-  singleLogoutService?: Array<{ Binding: string, Location: string }>;
+  assertionConsumeService?: Array<{ isDefault?: boolean, Binding: string, Location: string }>;
```

Comment:
Is changing the attribute name from assertionConsumerService to assertionConsumeService intentional?

Reply:
Nope, it's the typo. Thanks for pointing out.